<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Statistiche</class>
 <widget class="QMainWindow" name="Statistiche">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>490</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>490</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>490</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Statistiche</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnEsci">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>450</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Esci</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rBtnElencoPrivato">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Privato</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxLettera">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>A</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>D</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>E</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>F</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>G</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>H</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>I</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>J</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>K</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>L</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>M</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>N</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>O</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>P</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Q</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>R</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>T</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>U</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>V</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>W</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>X</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Y</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Z</string>
     </property>
    </item>
   </widget>
   <widget class="QTableView" name="tblElenco">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>711</width>
      <height>111</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="rBtnElencoAziende">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Azienda</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblLettera">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Mostra elenco cittadini che non hanno richiesto materiali ordinati per lettera:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCerca">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Cerca</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>711</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblIntestazioneGrafico">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>271</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Grafico dei materiali richiesti per mese:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxData">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>180</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2019</number>
    </property>
    <property name="maximum">
     <number>2090</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>180</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Anno</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
